ts/features/design-system/core/DSAccordion.tsx

```diff
@@ -16,22 +13,6 @@ import { VSpacer } from "../../../components/core/spacer/Spacer";
 
 export const DSAccordion = () => (
   <DesignSystemScreen title={"Accordion"}>
-    <Label>{"<IOAccordion />"}</Label>
-    <View style={[IOStyles.flex, { width: "100%" }]}>
-      <IOAccordion title={"Animated Accordion"}>
-        <AlphaChannel width={128} height={128} />
-      </IOAccordion>
-      <IOAccordion title={"Accordion without animation"} animated={false}>
-        <Analytics width={128} height={128} fill={IOColors.orange} />
-      </IOAccordion>
-      <IOAccordion title={"Accordion with a very very very very long text"}>
-        <>
-          <Analytics width={128} height={128} fill={IOColors.orange} />
-          <Analytics width={128} height={128} fill={IOColors.orange} />
-        </>
-      </IOAccordion>
-    </View>
-    <VSpacer size={40} />
```

Comment:
Why did you remove this section? I think `IOAccordion` should have some example in the DS.

Reply:
The `IOAccordion` was a replacement for the NativeBase one, but has never been used in the application. `RawAccordion' has been used once.

Now we have the new one from the DS 2.0. So basically we can get rid of `IOAccordion`